src/module/rules/rule-element/item-alteration/handlers.ts

```diff
@@ -38,6 +38,8 @@ class ItemAlterationHandler<TSchema extends AlterationSchema> extends fields.Sch
 
     operableOnSource: boolean;
 
+    phases: ItemAlterationPrepPhase[];
```

Comment:
Multiple?

Reply:
These are the supported ones, not the resolved ones. I realize now that its ambiguous, so I renamed it to supportedPhases.